lib/internal/url.js

```diff
@@ -290,6 +290,12 @@ class URLSearchParams {
     return `${this.constructor.name} {}`;
   }
 
+  get size() {
```

Comment:
This should probably be explicitly marked as enumerable, below in the `ObjectDefineProperties` block.